packages/fx-core/resource/backup-config-change-logs.md

```diff
@@ -0,0 +1,24 @@
+# Congratulations! You are now switching to another tenant or subscription. Configuration files for last provision/local debug are saved successfully.
+
+You are now using a Microsoft 365 account or an Azure subscription (applies only when provisioning in a remote environment of a project requiring cloud resources) that is different from what you used previously in the selected environment for provision or local debugging. We have backed up the configuration files which are used for or generated from previous provision or local debug. For more information, you can check this [doc](https://aka.ms).
+
```

Comment:
The aka link isn't created?

Reply:
Yes, will be created after this PR is completed, and then update the link

Reply:
Created link. but the link will only work after this PR is checked in.